packages/nimbus/src/components/money-input/money-input.tsx

```diff
@@ -122,28 +122,23 @@ export const MoneyInputComponent = (props: MoneyInputProps) => {
     placeholder = "0.00",
     autoFocus,
     size,
-    "aria-label": ariaLabel,
-    "aria-labelledby": ariaLabelledBy,
     ...restProps
   } = props;
 
   // Generate IDs
-  const generatedId = useId();
-  const id = providedId || generatedId;
+  const defaultGroupId = useId();
+  const groupId = id ?? defaultGroupId;
```

Comment:
The semantic group also pays off for having a framework to name things - the `id` goes on the group, so does `aria-label`, `aria-labelledby`, etc, etc. They are then associated with the child inputs in the group (thx react-aria group context)